lib/server/services/eosReport/EosReportService.js

```diff
@@ -113,7 +115,7 @@ class EosReportService {
             issuesLogEntries: shiftIssues,
             lhcTransitions: reportCreationRequest.lhcTransitions,
             shiftFlow: reportCreationRequest.shiftFlow,
-            infoFromPreviousShifter: reportCreationRequest.infoFromPreviousShifter,
+            infoFromPreviousShifter: infoFromPreviousShifter.errorMessage ? '-' : infoFromPreviousShifter.value ?? '',
```

Comment:
Maybe you should coalesce to a '-` in both case. So basically this can be that:

```suggestion
            infoFromPreviousShifter: !infoFromPreviousShifter.errorMessage && infoFromPreviousShifter.value || '-',
```

Reply:
Done